.github/workflows/release.yaml

```diff
@@ -35,6 +24,11 @@ jobs:
       arm64_image: ${{ steps.ghcr-images.outputs.arm64_image }}
       s390x_image: ${{ steps.ghcr-images.outputs.s390x_image }}
       s390x_digest: ${{ steps.ghcr-images.outputs.s390x_digest }}
+    env:
+      TAG: ${{ github.ref_name }}
+      REGISTRY: ghcr.io
+      USERNAME: ${{ github.actor }}
+      ORG: rancher-sandbox
```

Comment:
can't this be left as is and defined as top-level vars to be used in both workflows instead of passing them to each workflow?

Reply:
Sounds reasonable to me, thanks